test/api/client/utils/BaseApi.ts

```diff
@@ -26,16 +26,6 @@ export default class BaseApi {
       t0 = performance.now();
       // Execute with Axios
       httpResponse = await axiosInstance(httpRequest);
-      // Debug
-      // console.log('HTTP Request ====================================');
-      // console.log(httpRequest.baseURL);
-      // console.log(httpRequest.url);
-      // console.log(httpRequest.method);
-      // console.log(httpRequest.data);
-      // console.log(httpResponse.status);
-      // console.log(httpResponse.statusText);
-      // console.log(httpResponse.data);
-      // console.log('====================================');
```

Comment:
I put that in comment for debugging, don't remove them

Reply:
Then add a tunable to conditionally enable debug code path.